### ISSUE TITLE
toPlural removes punctuation after nouns

### ISSUE THREAD

**Issue by dqnykamp** [spencermountain/compromise#605]
var doc=nlp("cat, boy, and chair");
    console.log(doc.nouns().toPlural().all().out())

Outputs

    cats boy, and chairs

Pluralizing cat to cats removed the comma after it.  But, for a word like boy (which for some reason isn't pluralized), the comma is retained.

**Comment by spencermountain** [comment]
hey thanks Duane, that definetly looks like a bug.

Let me know if this is a priority. I'm half-way through [v12](https://medium.com/@spencermountain/compromise-in-2019-e7d0ca703320) and this includes a full re-write of our punctuation logic, so it should be fixed in the next major version.

cheers

**Comment by dqnykamp** [comment]
We can work around it for now.

**Comment by spencermountain** [comment]
hey Duane, got a fix for this in `12.2.0`, which I'll release later this afternoon
```js
let doc = nlp('cat, boy, and chair')
doc.nouns().toPlural()
console.log(doc.text())
//cats, boys, and chairs
```
thanks for your patience
